docs/topics/rn-new-features.adoc

```diff
@@ -16,9 +16,9 @@ To execute the rules, use the target `jakarta-ee`.
 
 .OpenRewrite recipe support
 
-link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can execute OpenRewrite recipes by using the link:{ProductDocUserGuideURL}[{ProductShortName} CLI].
+link:https://docs.openrewrite.org/[OpenRewrite] automates large-scale, distributed source code refactoring. You can execute OpenRewrite recipes by using the link:{ProductDocUserGuideURL}/getting_started#execute_cli-guide[{ProductShortName} CLI].
```

Comment:
execute > run.

Reply:
The full link is: https://access.redhat.com/documentation/en-us/migration_toolkit_for_applications/5.2/html/cli_guide/getting_started#execute_cli-guide

"execute" is part of the link's path. Will the renaming of files in this PR change this path name?

Reply:
Honestly, I would not bother linking to the CLI guide.